src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15559.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue15559 : _IssuesUITest
+	{
+		public override string Issue => "[iOS] Vertical layout content of label is truncated when a width request is set in the layout";
+
+		public Issue15559(TestDevice device) : base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.Layout)]
+		public void LabelDisplayWithoutCroppingInsideVerticalLayout()
+		{
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
The Windows snapshot is pending, already available in the latest build.
![image](https://github.com/user-attachments/assets/238eaa1d-cfab-4552-bd67-bc3cdf21c62a)

Reply:
Hi @jsuarezruiz ,
Included the snapshot for the Windows platform. Could you please check and let me know?